isxbot2_interface: true 
task_type: "jump"


